modules/navier_stokes/include/base/NSFVBase.h

```diff
@@ -721,6 +725,11 @@ NSFVBase<BaseType>::validParams()
       "'porous_medium_treatment' is enabled, the coefficients already contain a velocity "
       "multiplier but they are not multiplied with density yet!");
 
+  params.addParam<bool>(
+      "standard_friction_formulation", true, "Flag to enable standard friction formulation");
+  params.addParam<bool>(
+      "use_superficial", true, "Flag to enable the use of superficial fluid values");
```

Comment:
In the action you won't need this. This will be defined by the `porous_medium_treatment=false or true`.

Reply:
done